index.html

```diff
@@ -86,6 +86,285 @@ <h1>Support</h1>
           <h1>Privacy</h1>
```

Comment:
@Gbemisola19 Take a look at this, you can only have one `<h1>` tag in your project